packages/skygear-react-native/src/index.ts

```diff
@@ -72,8 +72,15 @@ async function uploadForm(
       form.append(name, value);
     }
   }
+
+  const contentType = (req.headers || {})["content-type"];
```

Comment:
This is a breaking change on RN SDK API.

I see you mentioned 'On iOS, content type sometimes can be inferred automatically' in #579, do you know when would the inference fail?

Reply:
My previous statement is wrong. On iOS, [content-type is not required](https://github.com/facebook/react-native/blob/master/Libraries/Network/RCTNetworking.mm#L114). So the inference is done by the server.

Reply:
The expected usage is the developer provide the correct content-type so when the asset is accessed, the content-type is set correctly. The problem is that the android implementation requires content-type to be set at client side.

Reply:
Sorry I don't quite understand. Do you mean this?
- On iOS, `Content-Type` does not need to be provided at client side; it would be inferred at server-side.
- On Android, `Content-Type` is needed at client side; server side does not infer the content type.

Reply:
Yes.